src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements Item {
+    public static final int MAX_VALUE = 50;
+    private String color;
```

Comment:
Lets make it of type `Color` since we already have an enum for it, its is preferred to use it instead of String